### PR TITLE
Build Instruction Updates

### PR DESCRIPTION
**Pull request by Jmeyer1292**
Updates build instructions and wstool stuff.

@woodaaron Please review.

### CODE REVIEW

**Review by woodaaron — CHANGES_REQUESTED** [review]
Motoman is not needed for Godel.
ros-industrial/industrial_core is now at kinetic-devel

**Comment by gavanderhoorn** [comment]
@woodaaron wrote:
> Motoman is not needed for Godel.

All the `*_sia20d_*` packages are dependent on `motoman_sia20d_support` fi.

**Comment by woodaaron** [comment]
@gavanderhoorn wrote:
> All the `*_sia20d_*` packages are dependent on motoman_sia20d_support fi.

Fair enough. Godel will build without motoman in the build directory, and the application notes on the readme all reference the abb_irb2400 (which I use). I assumed Motoman was a legacy piece.

**Comment by Jmeyer1292** [comment]
@gavanderhoorn @woodaaron The motoman is a legacy at the moment. We should remove it (and all the dead code still referencing it) or move it in a different PR.

**Comment by Jmeyer1292** [comment]
@woodaaron I have pointed the `industrial_core` repo in the rosinstall script to point at the main one. 

I have not touched motoman dependencies. I will do it in another thread.

File: godel.rosinstall [hunk 1]
```diff
@@ -1,10 +1,15 @@
...
     version: indigo-devel}
-- git: {local-name: abb, uri: 'https://github.com/ros-industrial/abb.git',
-    version: indigo-devel}
+
+# Custom abb until Kinetic release happens
+- git: {local-name: abb, uri: 'https://github.com/Jmeyer1292/abb.git',
+    version: kinetic-devel}
+
+- git: {local-name: industrial_core, uri: 'https://github.com/ros-industrial/industrial_core.git',
+    version: kinetic-devel}
```
Comment (gavanderhoorn) [inline godel.rosinstall:15]: Do you have a particular need to build `industrial_core` from source? If it's been released `rosdep` should just resolve the dependency to the binaries ..

Comment (Jmeyer1292) [reply godel.rosinstall:15]: Sure. I just tried to update and install ros-kinetic-industrial-core and it's not there yet. I will update it when these are released.

Do you have any good resources on the differences between all of the different ros dependency tools? `rosdep` vs `rosinstall` vs `wstool`? I'm slowly starting to care about being able to on-board others easily and would like to get this sorted out.

**Review by woodaaron — APPROVED** [review]

**Pull request merged**